scripts/collisions.js

```diff
@@ -59,26 +66,36 @@ const worker = (methodName, done) => {
   const stats = new Stats()
   let hasCollided = false
   let sum = 0
+  let startTime = Date.now()
+  let numRuns = 0
 
-  const isComplete = () => {
-    const moe = stats.length > 2
+  const computeMoe = () => stats.length > 2 && stats.amean() > 0
       ? stats.moe() / stats.amean()
       : null
 
-    return stats.length >= MIN_RUNS && (moe != null && (moe <= LO_MOE || (moe <= HI_MOE && sum >= MAX_SUM)))
+  const computeDuration = () => Date.now() - startTime
+
+  const isComplete = () => {
+    const duration = computeDuration()
+
+    const moe = computeMoe()
+    return numRuns >= MIN_RUNS &&
+      ((duration >= MAX_DURATION) ||
```

Comment:
If we've reached the minimum number of runs, and it took longer than `MAX_DURATION` to get there, we don't worry about the other conditions.